user/libpinyin/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "libpinyin"
+pkgver = "2.9.92"
```

Comment:
this is the wrong version, it should be 2.8.1 and the newer versions ignored in update.py, with the `ignore =` i gave you

Reply:
Never mind, just leave it anyway. Latest version of ibus-libpinyin, which will be commited later, requires some features from it. that's a pre-release as well, but i never encountered an issue these days.

Reply:
no, you are not supposed to package prereleases, including for ibus-libpinyin. for ibus-libpinyin, you'll need 1.15.8 and not 1.15.92

Reply:
Got it, just never package pre-releases, only stable/release software are accepted, right?

Reply:
generally yes

there's sometimes an exception if some piece of software is useful and would be added, but was also abandoned so the latest git commit might just have fixes not present in the last release, etc. so usually never, but e.g. libkkc exists so there was no reason to not just use the latest commit